environment.yml

```diff
@@ -23,11 +23,12 @@ dependencies:
 - freetype=2.7=vc9_1
 - freexl=1.0.2=vc9_2
 - functools32=3.2.3.2=py27_1
-- gdal
+- geos
+- gdal=2.1.3=py27_vc9_7
 - geopandas>=0.2.1
-- geos>=3.5.1
 - hdf4=4.2.12=vc9_0
 - hdf5>=1.8.18
+- h5py
```

Comment:
I am not sure if we need this. I am using pandas.to_hdf and pandas.read_hdf to use hdf5

Reply:
We don't need it specifically... it seems to be included in one of our dependencies. I'm removing it from the list.